artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/DockerEngine_DockerEngine-Interface-w1/source/src/main/java/org/opentosca/artifacttemplates/DockerEngineInterfaceDockerEngineEndpoint.java

```diff
@@ -332,6 +332,12 @@ public void onNext(final BuildResponseItem item) {
                 }
             }
 
+
+            final Boolean privileged = request.getPrivilegedMode() != null && request.getPrivilegedMode().equals("true");
```

Comment:
I would use Boolean.parseBoolean() instead of equals("true) here to be a bit more inclusive regarding case sensitivity, etc.

Reply:
Also remove the additional empty line above :-)